traveler_app/src/main/AndroidManifest.xml

```diff
@@ -13,6 +15,9 @@
             android:supportsRtl="true"
             android:theme="@style/AppTheme"
             android:usesCleartextTraffic="true">
+        <meta-data
+                android:name="com.google.android.geo.API_KEY"
+                android:value="AIzaSyD1-fBPgL__ugKuoMtWHs26Ko0B4Pr6PGc"/>
```

Comment:
This should not be committed to code..Use BuildConfig instead

Reply:
✓